src/location-distance-utils.h

```diff
@@ -24,6 +24,8 @@
 
 G_BEGIN_DECLS
 
+float lng_scale(double lat);
```

Comment:
why is that exported? Is there any user of it outside location-distance-utils.c

Reply:
Ultimately yes, for terrain and bearing calculations, it shouldn't be long to add them.